pandas/core/indexes/base.py

```diff
@@ -5135,9 +5135,30 @@ def delete(self, loc):
         """
         Make new Index with passed location(-s) deleted.
 
+        Use array of integer as loc parameter to delete multiple locations.
+
+        Parameters
+        ----------
+        loc : int
+            Location of item(-s) which will be deleted.
+
         Returns
         -------
-        new_index : Index
+        Index
+            New Index with passed location(-s) deleted.
+
+        See Also
+        --------
+        numpy.delete : Delete any rows and column from NumPy array (ndarray).
+
+        Examples
+        --------
+        >>> idx = pd.Index(['a', 'b', 'c'])
+        >>> idx.delete(1)  # Deleting 'b'
```

Comment:
I think you can remove the comment, the example is good enough to be clear that we're deleting b